src/table.ts

```diff
@@ -683,16 +683,20 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       rowsLimit = options.limit;
     }
 
-    const userStream = through.obj();
+    const userStream = through.obj((rowData, enc, next) => {
+      next(null, rowData);
+    });
     const end = userStream.end.bind(userStream);
     userStream.end = () => {
+      rowStream.unpipe(userStream);
```

Comment:
Just to double check,  do we need to make sure `rowStream` is defined first?

Reply:
it might be good to add a check:

```
if (rowStream) rowStream.unpipe(userStream);
```

There is homegrown retry logic, such that `rowStream` isn't defined until a closure is called, and might be redefined if an error occurs, so the definition of `rowStream` can't easily happen above `userStream`.

Reply:
I added a check. This is practically not an issue, but in theory it could happen.